ophyd/ophydobj.py

```diff
@@ -74,7 +126,15 @@ class OphydObject:
     name
     '''
 
+    # Any callables appended to this mutable class variable will be notified
+    # one time when a new instance of OphydObj is instantiated. See
+    # OphydObject.add_instantiation_callback().
+    __instantiation_callbacks = []
     _default_sub = None
+    # This is set to True when the first OphydObj is instiated. This may be of
+    # interest to code that adds something to instantiation_callbacks, which
+    # may want to know whether it has already "missed" any instances.
+    __instantiated = False
```

Comment:
maybe name this `__any_instantiated`?  `__instatiated` feels like it is a `per-instance` name...